config/nrfconnect/chip-module/Kconfig

```diff
@@ -144,6 +144,22 @@ config CHIP_DEBUG_SYMBOLS
 	help
 	  Enables building the application with debug symbols.
 
+config CHIP_LOG_VERIFY_OR_DIE
+	bool "Log source code location on VerifyOrDie failure"
+	help
+	  Enables the feature to log the file name and line number where the Matter
+	  stack calls the VerifyOrDie macro with the condition evaluating to false.
+
+config CHIP_LOG_FILE_NAME
+	bool "Log file name instead of entire file path"
+	default y
```

Comment:
Let me know if it's OK to enable this be default. This allows to save a few Ks for debug builds with OpenThread assertions etc.

Reply:
From my point of view, it is ok to have it enabled by default.

Reply:
LGTM 👍